.clang-tidy

```diff
@@ -13,6 +13,11 @@ Checks: >
     -bugprone-forward-declaration-namespace,
     -bugprone-sizeof-expression,
     -bugprone-throw-keyword-missing,
+    -bugprone-chained-comparison,
```

Comment:
Many of these warnings seem to be useful - I'll create an issue to fix some of them before merging it.

Reply:
https://github.com/Project-OSRM/osrm-backend/issues/6928